src/components/people/PersonPicture.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * @flow
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { faUser } from '@fortawesome/pro-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Map } from 'immutable';
+
+import { FullyQualifiedNames } from '../../core/edm/constants';
+
+const {
+  PERSON_MUGSHOT_FQN,
+  PERSON_PICTURE_FQN,
+} = FullyQualifiedNames.PROPERTY_TYPE_FQNS;
+
+const SizeMap = {
+  sm: {
+    font: '21px',
+    pixels: '45px',
+  },
+  md: {
+    font: '36px',
+    pixels: '70px',
+  },
+  xl: {
+    font: '100px',
+    pixels: '200px',
+  },
+};
+
+const PictureWrapper = styled.div`
+  border-radius: ${({ round }) => (round ? '50%' : 0)};
+  display: flex;
+  height: ${({ size }) => SizeMap[size].pixels};
```

Comment:
This is all well and good, but have you considered using the `getStyleVariation` util in LUK?

If so, was there anything you found lacking?

If not, the API would be as follows. While it does mean you have to define your font variation separately from your size variation, they aren't used together in the same component anyway. This way you can avoid repeating the same anonymous function in multiple places.

```
import styled, { css } from 'styled-components';
import { StyleUtils } from 'lattice-ui-kit';

const { getStyleVariation } = StyleUtils;

const  getPictureDimension = getStyleVariation('size', {
  css`
    height: 45px;
    width: 45px;
  `,
  ...
});

const PictureWrapper = styled.div`
  ...
  ${getPictureSize}
`;

// you can also use it without defining a whole css block
const getFontSize = getStyleVariation('size', {
  sm: '21px',
  md: '36px',
  xl: '100px'
});

const UserIconWrapper = styled.div`
  ...
  font-size: ${getFontSize};
`;

```